manifests/rhoai/addon/apps/nvidia/gpu-quickstart.yaml

```
@@ -39,19 +39,19 @@ spec:
       description: |-
         Prerequisites:
 
-        - You have no notebook servers running in Red Hat OpenShift AI.
+        - You have no workbenches running in Red Hat OpenShift AI.
 
         1. Navigate to the Red Hat OpenShift AI dashboard.
-        2. Click **Launch** on the Jupyter card to open the **Start a notebook server** page.
+        2. Click **Launch** on the Jupyter card to open the **Start a workbench** page.
```

Oh I just got one question, should we update this to match the updates you are making for the standalone workbench ticket?
Something like
```suggestion
        2. Click **Launch application** on the Jupyter card to open the **Start a basic workbench** page.
```
Same questions to all the other changes when it refers to the standalone workbench page.

---

Yes, good catch